test/integration/features/config.feature

```diff
@@ -0,0 +1,90 @@
+            | crc config set warn-check-libvirt-running true       | crc config set warn-check-libvirt-running false       | warn-check-libvirt-running       |
+            | crc config set warn-check-crc-network true           | crc config set warn-check-crc-network false           | warn-check-crc-network           |
+            | crc config set warn-check-crc-network-active true    | crc config set warn-check-crc-network-active false    | warn-check-crc-network-active    |
+
+# SKIP
+
+    Scenario Outline: CRC config checks (skips)
+        When executing "<check>" succeeds
+        Then file "crc.json" exists in CRC home folder
+        And "JSON" config file "crc.json" in CRC home folder contains key "<property>" with value matching "true"
+        When executing "crc setup" succeeds
+        Then stdout should contain "Skipping above check ..."
+        When executing "<nocheck>" succeeds
+        Then "JSON" config file "crc.json" in CRC home folder contains key "<property>" with value matching "false"
+        When executing "crc setup" succeeds
+        Then stdout should not contain "Skipping above check ..."
+        
+        Examples:
+            | check                                                | nocheck                                               | property                         |
+            | crc config set skip-check-virt-enabled true          | crc config set skip-check-virt-enabled false          | skip-check-virt-enabled          |
+            | crc config set skip-check-kvm-enabled true           | crc config set skip-check-kvm-enabled false           | skip-check-kvm-enabled           |
+            | crc config set skip-check-libvirt-enabled true       | crc config set skip-check-libvirt-enabled false       | skip-check-libvirt-enabled       |
+            | crc config set skip-check-libvirt-running true       | crc config set skip-check-libvirt-running false       | skip-check-libvirt-running       |
+            | crc config set skip-check-crc-network-active true    | crc config set skip-check-crc-network-active false    | skip-check-crc-network-active    |
+            | crc config set skip-check-libvirt-installed true     | crc config set skip-check-libvirt-installed false     | skip-check-libvirt-installed     |
+            | crc config set skip-check-libvirt-driver true        | crc config set skip-check-libvirt-driver false        | skip-check-libvirt-driver        |
+            | crc config set skip-check-crc-network true           | crc config set skip-check-crc-network false           | skip-check-crc-network           |
+            | crc config set skip-check-crc-dnsmasq-file true      | crc config set skip-check-crc-dnsmasq-file false      | skip-check-crc-dnsmasq-file      |
+            | crc config set skip-check-user-in-libvirt-group true | crc config set skip-check-user-in-libvirt-group false | skip-check-user-in-libvirt-group |
```

Comment:
Again these all will work with linux so use a build tag for different operating system.